src/directives/formly-form.js

```diff
@@ -16,7 +16,7 @@ module.exports = ngModule => {
       restrict: 'E',
       template: function(el, attrs) {
         /* jshint -W033 */ // this because jshint is broken I guess...
-        var rootEl = attrs.rootEl || 'ng-form';
+        var rootEl = attrs.rootEl || 'form';
```

Comment:
You're absolutely right, I'll close this PR and will open a new one with these lines reverted.